third_party/terraform/tests/resource_storage_hmac_key_test.go

```diff
@@ -0,0 +1,68 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestStorageHmacKey_update(t *testing.T) {
+	t.Parallel()
+
+	saName := saName()
+	bucketName := testBucketName()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckStorageHmacKeyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testGoogleStorageHmacKeyBasic(saName, bucketName),
+			},
+			{
+				ResourceName:      "google_storage_hmac_key.default",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func saName() string {
+	return fmt.Sprintf("", "tf-test-bucket", acctest.RandInt())
```

Comment:
You can just inline this- I think you want the first string to be `"%v%v"` as well.

Reply:
Also, `acctest.RandString(10)` instead of `RandInt`